conboarding.md

```diff
@@ -39,6 +39,17 @@ How much does it cost to run things?
 
 
 1. Send desired login name and .pub portion of the SSH key to Yarik for development box(es): smaug, typhon, etc
+   1. login to `smaug`: `ssh -i /path/to/key me@smaug.dartmouth.edu -p
+      $SOME_PORT`
+   1. login to `typhon`: `ssh -i /path/to/key me@typhon.dartmouth.edu -p
+      $SOME_PORT`
+      Note: When loging in from campus (use `eduroam`), SSH does not always work on
+      `typhon`. Instead it is recommended to use `ssh-agent` and forward the authentication connection using `-A`
+         1. (Assuming `ssh-agent` is running) `ssh-add /path/to/key`
```

Comment:
FWIW I use/recommend to time out keys, so do smth like
```suggestion
         1. (Assuming `ssh-agent` is running) `ssh-add -t 3600 /path/to/key`
```
to time out in an hour, and thus be sure that if laptop is "posessed" by someone later one, key would likely no longer be available decrypted in the memory